sds_data_manager/sds_cognito_stack.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+import aws_cdk as cdk
+import aws_cdk.aws_cognito as cognito
+import aws_cdk.aws_iam as iam
+import aws_cdk.aws_lambda as lambda_
+import aws_cdk.aws_lambda_python_alpha as lambda_alpha_
+from aws_cdk import (
+    Stack,
+)
+from constructs import Construct
+
+
+class SdsCognitoStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        sds_id: str,
+        initial_email: str = "",
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        ########### INIT
+        initial_user_context = initial_email
+
+        ########### IAM POLICIES
```

Comment:
```suggestion
```
Your variable names are good already, I don't think we need comments reiterating the variable names.